ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiConfigManagementService.scala

```diff
@@ -57,11 +59,13 @@ private[apiserver] final class ApiConfigManagementService private (
   override def bindService(): ServerServiceDefinition =
     ConfigManagementServiceGrpc.bindService(this, executionContext)
 
-  override def getTimeModel(request: GetTimeModelRequest): Future[GetTimeModelResponse] =
+  override def getTimeModel(request: GetTimeModelRequest): Future[GetTimeModelResponse] = {
+    logger.info("Getting time model")
```

Comment:
Does this log line bring enough in its current form? Should it be `info` ?

Reply:
That's a good question. There's no information in the request itself that is suitable for putting in the `LoggingContext`. We could create a correlation id at this point but according to @stefanobaghino-da 's comment we're fine with this simple logging. Please see [this thread](https://github.com/digital-asset/daml/pull/8676#discussion_r568441915).

Reply:
those simple get* requests have very little meat on the bone. They are also very shallow i.e. don't reach to committer. It is ok to log little info here.